Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Build the UI
+FROM node:18.17.0-bullseye-slim AS nodework
+WORKDIR /app
+COPY ui/ .
+RUN npm install
+RUN npm run build
+
+# Build the Go binary
+FROM golang:1.22 AS gowork
+WORKDIR /app
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY . .
+COPY --from=nodework /app/build ui/build
+RUN go build -o /app/main .
+
+
+# Run the binary in a alpine container
+FROM ubuntu:22.04
```

Comment:
Alpine image would be a nice to have over ubuntu, any chance if this could be done? I believe that an alpine image should be lighter than ubuntu?